### PR TITLE
PR: Issue 194

### PR DESCRIPTION
**Pull request by SamFritz**
Takes care of [issue 194](https://github.com/archivesunleashed/archivesunleashed.org/issues/195) 

Adds in AU Cohort 1 material to project website:

- [x]  New Cohorts page (@ianmilligan1 set this up)
- [x]  Add in banner/button/link on home page to draw attention to Cohorts

Page Content to include:

- [x]  Information about the event
- [x]  link to download PDF for application (just making some last minute tweaks)
- [x]   important dates
- [x]  organizers
- [x]   evaluation criteria

Let's hold off on publishing Cohort pages once we open applications (Mid-Feb)

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -5,17 +5,25 @@ type: index
...
 
 ## The Project
 
-The Archives Unleashed project aims to make petabytes of historical internet content accessible to scholars and others interested in researching the recent past. Supported by [The Andrew W. Mellon Foundation](https://mellon.org/), we develop web archive search and data analysis tools to enable scholars, librarians and archivists to access, share, and investigate recent history since the early days of the World Wide Web.
+The Archives Unleashed project aims to make petabytes of historical internet content accessible to scholars and others interested in researching the recent past. Our team develops web archive search and data analysis tools to enable scholars, librarians and archivists to access, share, and investigate recent history since the early days of the World Wide Web.
+
+
+Supported by a new grant by [The Andrew W. Mellon Foundation](https://mellon.org/), Archives Unleashed is partnering with our colleagues at the [Internet Archive](https://archive.org). We will be integrating the Cloud with [Archive-It service](https://archive-it.org) to ensure the project’s long-term sustainability, as well as enhance usability and accessibility of web archives.
```
Comment (ianmilligan1) [inline content/_index.md:20]: I suppose we can drop "new grant" at this period (how time flies)

Maybe just "Supported by [The Andrew W. Mellon Foundation](https://mellon.org/), Archives Unleashed is..."

Comment (ianmilligan1) [reply content/_index.md:20]: also should add a "the" before "Archive-It service"

File: content/_index.md [hunk 2]
```diff
@@ -5,17 +5,25 @@ type: index
...
+[Read more about the next stage of the project here](https://uwaterloo.ca/arts/news/archives-unleashed-project-scales-archive-it-better).
+
+
+## Acknowledgments
 
+The work of this project is made possible thanks to the generous support of financial and in-kind support from the following institutions:
 
-We invite you to follow the links at left to engage with our [community](/get-involved) and explore our project and tools further!
\ No newline at end of file
+![funding agencies](/images/logo-AU-funding.png)
```
Comment (ianmilligan1) [inline content/_index.md:29]: Fancy!

File: content/cohorts/index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## About
+
+Big Data, in the form of born-digital historical sources, is reshaping the humanities and social sciences. The sheer amount of cultural information that is generated and, crucially, preserved every day in electronic form, presents exciting new opportunities for historians, political scientists, sociologists, linguistics, computer scientists, and other scholars. Much of this information is captured within web archives, created by organizations such as the Internet Archive and many other libraries.
+
+The Archives Unleashed project is pleased to announuce the launch of the **Cohort Program**, which will facilitate research engagement with web archives. Cohorts will engage in a year-long collaboration, while receiving mentorship and support from the Archives Unleashed Team.
+
+## Program Details
+
+We are looking to select **five cohorts** in the July 2021 to June 2022 period who will receive the following support:
```
Comment (ianmilligan1) [inline content/cohorts/index.md:17]: let's put "up to" five cohorts (classic hedging of bets)

File: content/cohorts/index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+* **Participate** in cohort events which will provide an introduction to web archives and an opportunity to workshop draft work in a supportive environment.
+	* Opening event: virtual [July 2021]
```
Comment (ianmilligan1) [inline content/cohorts/index.md:20]: Let's specify that it's virtual due to COVID-19

File: content/cohorts/index.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+	* Closing event: Internet Archive, San Francisco, California, United States [May 2022]
```
Comment (ianmilligan1) [inline content/cohorts/index.md:21]: Can we put some tentative language in there, i.e. Tentatively May 2022

Tentative in that things could shift due to logistics, _and/or_ public health measures. Knock on wood not the latter.

File: content/cohorts/index.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+* **Bi-weekly mentorship** and support meetings with the Archives Unleashed team;
```
Comment (ianmilligan1) [inline content/cohorts/index.md:22]: I always get confused with 'bi-weekly' as it seems to have two different meanings!

Let's bring mentorship/support meetings together to `bi-monthly mentorship meetings`

File: content/cohorts/index.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+* **Computational support** including storage and computational; and
+* **Funding** of $11,500 CAD in to support project work and travel to the closing event.
```
Comment (ianmilligan1) [inline content/cohorts/index.md:24]: remove `and travel to the closing event` -> we can get separate support on closing event travel 👍 

so the $11,500 can be relatively unrestricted

File: content/cohorts/index.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+It is expected that all teams will **prepare a journal-length article submission** as part of this project, which will be developed throughout the term and workshopped at the closing May 2022 event at the Internet Archive.
+
+## The Cohorts
+
+**The Cohorts**: can be disciplinary or interdisciplinary and should include no more than five individuals. Graduate students, postdoctoral fellows, and those in precarious or sessional positions are welcome to apply; however, an administrative lead must be in place at an institution eligible to administer and receive grant funds. This would regularly be a full-time faculty member.
```
Comment (ianmilligan1) [inline content/cohorts/index.md:30]: can probably drop the `:` after The Cohorts

File: content/cohorts/index.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+**Important Dates**
+
+* Applications Due: 31 March 2021
+* Notifications: 15 April 2021
+* Projects Begin: 1 July 2021
+* Projects End: 30 June 2022
+---
+
+Prepare you submission package using the **Cohort application**, to address the following areas:
```
Comment (ianmilligan1) [inline content/cohorts/index.md:46]: I assume Cohort Application will link to the PDF?

File: content/cohorts/index.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+* Project name and list of all applicants;
+* Identify administrative lead (needs to be eligible to administer and receive grant funds);
+* CV for each named applicant (max 2 pages each); and
+* Project proposal:
+	* **Project Summary**: describe the research problem you aim to approach with web archives;
```
Comment (ianmilligan1) [inline content/cohorts/index.md:52]: Should we put a word limit on this? Def. no more than 500 words. Maybe we can chat at the team meeting?

File: content/cohorts/index.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+	* **Dataset**: identify the web archive collection you plan to use to explore your project; and
+	* **Timeline**: provide an brief overview of how you will conduct the project.
```
Comment (ianmilligan1) [inline content/cohorts/index.md:54]: Def. no more than 250 words or so here.

**Review by ianmilligan1 — CHANGES_REQUESTED** [review]
First pass - this looks great! Thanks @SamFritz. Some relatively minor suggestions here.

File: content/cohorts/index.md (hunk 6 above)
Comment (SamFritz) [reply content/cohorts/index.md:22]: ah yes! thank you!

File: content/cohorts/index.md (hunk 9 above)
Comment (SamFritz) [reply content/cohorts/index.md:46]: Yep! I will add in PDF, once language is tweaked/fix the fill-in box descriptions

File: content/cohorts/index.md (hunk 10 above)
Comment (SamFritz) [reply content/cohorts/index.md:52]: added to agenda. Also updated this to read (max. 500 words)....which can be changed after group chat.

File: content/cohorts/index.md (hunk 11 above)
Comment (SamFritz) [reply content/cohorts/index.md:54]: again, put place holder text at end of datasets/timeline to reach (max. 250 words)

**Review by ianmilligan1 — APPROVED** [review]
Looks great - we'll bring some of those little questions to the standup meeting on Thursday!

**Comment by SamFritz** [comment]
Will be adding in a new section to address examples of scholarship for inspiration: 

- Trevor Owens + Grace Thomas:  [The invention and dissemination of the spacer gif: implications for the future of access and use of web archives](https://doi.org/10.1007/s42803-019-00006-8)
  - spacer.gif (Nick's [blog post](https://news.archivesunleashed.org/geocities-and-the-spacer-gif-6a0f70e7ad5d) about their project)
- Matt Weber's [Assessing Local Journalism: News Deserts, Journalism Divides, and the Determinants of the Robustness of Local News](https://dewitt.sanford.duke.edu/wp-content/uploads/2018/08/Assessing-Local-Journalism_100-Communities.pdf)
- Datathon Projects:
  - Team Wildfyer (Vancouver)
  - Team Kompromat (Washington)
  - Team DC Punk (Washington)
  - Team Composers (NY)

File: content/cohorts/index.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+
+## Program Details
+
+We are looking to select up to **five cohorts** in the July 2021 to June 2022 period who will receive the following support:
+
+* **Participate** in cohort events which will provide an introduction to web archives and an opportunity to workshop draft work in a supportive environment.
+	* Opening event: virtual, due to COVID-19 [July 2021]
+	* Closing event: Internet Archive, San Francisco, California, United States [Tentatively May 2022]
+* **Bi-monthly mentorship** and support meetings with the Archives Unleashed team;
+* **Access to the Archive-It Research Cloud** to generate custom datasets; and
```
Comment (ianmilligan1) [inline content/cohorts/index.md:23]: 👍

File: content/cohorts/index.md [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+---
+Submissions will be evaluated on the following criteria:
+* demonstration of a feasible project that will make an impact on the scholar’s field of study;
+* the selection of specific web archives to help achieve that project;
+* suitability of the project team to learn and carry out planned activities; and
+* suitable documentation of a web archival collection that can support this research.
+
+## Project Inspiration
+
+Need some ideas to inspire your Cohort project? We've listed a few recent scholarly works and projects to showcase the variety of ways web archives can be explored, used, and visualized.
```
Comment (ianmilligan1) [inline content/cohorts/index.md:67]: "scholarly works and projects" -> maybe just "projects"?

Comment (ianmilligan1) [reply content/cohorts/index.md:67]: Maybe "Cohort project" -> "Cohort proposal"

File: content/cohorts/index.md [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+
+* [Assessing Local Journalism: News Deserts, Journalism Divides, and the Determinants of the Robustness of Local News](https://dewitt.sanford.duke.edu/wp-content/uploads/2018/08/Assessing-Local-Journalism_100-Communities.pdf) by Philip M. Napoli, Matthew Weber, Katie McCollough & Qun Wang
```
Comment (ianmilligan1) [inline content/cohorts/index.md:69]: Maybe put periods after the author lists? (getting pedantic here I know!)

File: content/cohorts/index.md [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+* [The invention and dissemination of the spacer gif: implications for the future of access and use of web archives](https://doi.org/10.1007/s42803-019-00006-8) by Trevor Owens & Grace Thomas
+* [GeoCities and the spacer.gif](https://news.archivesunleashed.org/geocities-and-the-spacer-gif-6a0f70e7ad5d) by Nick Ruest
```
Comment (ianmilligan1) [inline content/cohorts/index.md:71]: Maybe combine line 70 and 71; noting that Nick's piece is a sort of response of building on that research using our tools?

File: content/cohorts/index.md [hunk 16]
```diff
@@ -0,0 +1,86 @@
...
+
+**The Cohorts** can be disciplinary or interdisciplinary and should include no more than five individuals. Graduate students, postdoctoral fellows, and those in precarious or sessional positions are welcome to apply; however, an administrative lead must be in place at an institution eligible to administer and receive grant funds. This would regularly be a full-time faculty member.
+
+The University of Waterloo regards diversity as an integral part of academic excellence and is committed to accessibility for persons with disabilities. As such, we encourage applications from women, First Nations, Metis and Inuit peoples, persons with disabilities, members of diverse gender identities, and others who may contribute to the further diversification of ideas.
+
+**The Data**: With our recently announce integration with Archive-It, data available for use will primarily be institution web archive collections in Archive-it (https://archive-it.org). Additional datasets from the overall Wayback Machine and Internet Archive may also be available. We are happy to work with applicants to address any questions; please connect with [Samantha Fritz](mailto:samantha.fritz@uwaterloo.ca).
```
Comment (ianmilligan1) [inline content/cohorts/index.md:34]: Second Archive-it should be capitalized (i.e. Archive-It)

**Review by ianmilligan1 — CHANGES_REQUESTED** [review]
Looks great - just a few minor suggestions!

File: content/cohorts/index.md [hunk 17]
```diff
@@ -43,18 +43,17 @@ The University of Waterloo regards diversity as an integral part of academic exc
...
 ---
 
-Prepare you submission package using the **Cohort application**, to address the following areas:
+Prepare you submission package using the [**Cohort application**](/images/AU-cohort-application.pdf), which addresses the following areas:
 
 * Project name and list of all applicants;
-* Identify administrative lead (needs to be eligible to administer and receive grant funds);
-* CV for each named applicant (max 2 pages each); and
+* Identifies an administrative lead (needs to be eligible to administer and receive grant funds);
+* CV for each named applicant (max 2 pages each) and can be attached to the email or appended to the applicaiton PDF; and
```
Comment (ianmilligan1) [inline content/cohorts/index.md:50]: Just see a typo on "application" here!

File: content/cohorts/index.md [hunk 18]
```diff
@@ -21,7 +21,7 @@ We are looking to select up to **five cohorts** in the July 2021 to June 2022 pe
 	* Closing event: Internet Archive, San Francisco, California, United States [Tentatively May 2022]
 * **Bi-monthly mentorship** and support meetings with the Archives Unleashed team;
 * **Access to the Archive-It Research Cloud** to generate custom datasets; and
-* **Funding** of $11,500 CAD to support project work.
+* **Funding** of $11,500 CAD to support project work. Additional support will be provided for travel to the Internet Archives event.
```
Comment (ianmilligan1) [inline content/cohorts/index.md:24]: Internet Archives -> Internet Archive (singular)

**Review by ianmilligan1 — CHANGES_REQUESTED** [review]
Just two minor typos caught!

File: content/cohorts/index.md (hunk 17 above)
Comment (SamFritz) [reply content/cohorts/index.md:50]: oh my goodness! thank you for this catch!

**Pull request merged**